docs/aerodromes/classd/Bankstown.md

```diff
@@ -139,4 +161,14 @@ VFR aircraft require a level readback.
     <span class="hotline">**BK ADC** -> **SY TCU**</span>: "UJN, A010"
 
 !!! tip
-    Remember that IFR aircraft are only separated from other IFR or SVFR aircraft in class D. You should *generally* be able to issue a clearance for an approach and use other separation methods (visual separation, holding a departure on the ground) if separation is required with these aircraft.
\ No newline at end of file
+    Remember that IFR aircraft are only separated from other IFR or SVFR aircraft in class D. You should *generally* be able to issue a clearance for an approach and use other separation methods (visual separation, holding a departure on the ground) if separation is required with these aircraft.
+
+#### ADC 2 Online
+When ADC 2 is online, SY TCU may not be familiar with which controller owns what airspace. Best practice is to receive the coordination no matter what, and if it was meant for the other ADC controller, relay the coordination to them internally.
+
+### BK ADC Internal
+BK ADC must heads-up coordinate **all aircraft** transiting from one ADC controller to the other.
+
+!!! phraseology
+    <span class="hotline">**BK ADC 2** -> **BK ADC**</span>: "via TWRN, EWY for an overhead join"  
+    <span class="hotline">**BK ADC** -> **BK ADC 2**</span>: "EWY, A015"
```

Comment:
Can we incorporate Runway Dependency when using 29C/11C for departures / prohibition for opposite bases to adjacent runways  (29L/29C & 29C/29R)? Let me know if you need more info... just thought it'd be good for those keen

Reply:
Yeah could  you flesh this out a bit? Im a bit confused haha

Reply:
Gets complicated... happy to leave it out, but here it is;

RWY Dependency;
The distance between adjacent Runways (Centrelines) at Bankstown does not meet the standard required for Independent Runway Operations. Actual distance 66m(?). Required distance for Single Engines = 60m, Multi-Engine = 90m, Jet = 120m (off the top of my head).

Because Single Engine aircraft are independent (above), you can use the runways independently (ie at the same time). However, the moment a Twin/Jet is involved, you require a runway standard with the adjacent runway(s) in addition the the runway in use. 

Note; only applies for departures (an exemption exists for arrivals, causing opposite base issue, more below)

My proposed clause would probably say something like;

(Departures Section)
"Due to the close proximity of parallel runways at Bankstown, runways are NOT independent for multi-engine and Jet aircraft. Controllers MUST apply a runway standard to the Runway(s) immediately adjacent when departing Multi-Engined or Jet Aircraft

Note; Arriving aircraft are exempt from this runway dependency.

For example; A C510 (Jet) is ready runway 29C. The controller must apply a separation standard to ALL 3 runways simultaneously when clearing this acft for take-off. However, clearing an aircraft to land (not touch and go) whilst the C510 departs on an adjacent runway is permitted.


Opposite Base;

CASA instrument exists permitting independent arrivals to adjacent runways, however (due to a midair 20 years ago), aircraft cannot BOTH make the turn to final in close proximity. Controllers must instead ensure at least one of the aircraft is established on final prior to entering conflicts, AND mutual traffic is passed in the event someone passes thru final

Suggested clause; 

Arrivals;
Due to high risk of collision and poor field of view during turns to final, Do NOT permit a pair aircraft to turn final simultaneously and/or in close proximity when assigned adjacent runways (EG; 29R & 29C, or 29C & 29L). Controllers should stagger base turns or issue 'go arounds' to solve conflicts.
This does not apply to aircraft pairing assigned adjacent where an aircraft is already established on final AND traffic has been passed to BOTH aircraft.

Reply:
Yep all makes sense.

Would be a bit of a bigger job though, we currently don't publish those sep standards anywhere, so we'd need to add those in too, and include some stuff in the moodle. Maybe something for down the line?

Reply:
I'd be fine with that. Sep standard are already in place in other sections anyway, even if they aren't the smallest we can get (600m).